temporalio/exceptions.py

```diff
@@ -70,6 +70,17 @@ def __init__(
         self.run_id = run_id
 
 
+class ApplicationErrorCategory(IntEnum):
+    """Severity category for your application error. Maps to corresponding client-side logging/metrics behaviors"""
```

Comment:
May want a docstring on benign saying it no longer logs at the error/exception level and does not record metrics

Reply:
+1 to that

Reply:
added